corehq/apps/reports/v2/utils.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import dateutil
+import pytz
+from memoized import memoized
+
+from corehq.const import SERVER_DATETIME_FORMAT
+from corehq.util.dates import iso_string_to_datetime
+from corehq.util.timezones.conversions import PhoneTime
+from corehq.util.timezones.utils import get_timezone_for_user
+
+
+@memoized
+def get_timezone(request, domain):
+    if not domain:
+        return pytz.utc
+    else:
+        try:
+            return get_timezone_for_user(request.couch_user, domain)
+        except AttributeError:
+            return get_timezone_for_user(None, domain)
+
+
+def parse_date(date_string):
```

Comment:
This one also seems like it should go in `timezone.utils`

Reply:
fixed